pom.xml

```diff
@@ -268,7 +253,6 @@
          </plugin>
          <plugin>
             <artifactId>maven-scm-plugin</artifactId>
-            <version>${maven.scm.plugin.version}</version>
```

Comment:
My Maven is complaining, seem to be necessary: 
```console
Some problems were encountered while building the effective model for org.eclipse.hawkbit:hawkbit-core:jar:0.3.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-scm-plugin is missing. @ org.eclipse.hawkbit:hawkbit-parent:0.3.0-SNAPSHOT

It is highly recommended to fix these problems because they threaten the stability of your build.
For this reason, future Maven versions might no longer support building such malformed projects.
```